Origin: https://sources.debian.org/src/socat/1.8.0.2-1/debian/patches/03-Truncate/
Source-package: socat_1.8.0.2-1.debian.tar.xz (SHA256 04b361cd91884eda6ca9fe2890ca83a6abe73dd8ce75fe9927b2d8443af36123)

Error instead of warn on address truncation.

Forwarded: no
Author: Chris Taylor <ctaylor@debian.org>
--- socat-1.7.2.4.orig/xio-unix.c
+++ socat-1.7.2.4/xio-unix.c
@@ -79,7 +79,7 @@ xiosetunix(int pf,
 #ifdef WITH_ABSTRACT_UNIXSOCKET
    if (abstract) {
       if ((pathlen = strlen(path)) >= sizeof(saun->sun_path)) {
-	 Warn2("socket address "F_Zu" characters long, truncating to "F_Zu"",
+	 Error2("socket address "F_Zu" characters long, max length is "F_Zu"",
 	       pathlen+1, sizeof(saun->sun_path));
       }
       saun->sun_path[0] = '\0';	/* so it's abstract */
@@ -98,7 +98,7 @@ xiosetunix(int pf,
 #endif /* WITH_ABSTRACT_UNIXSOCKET */
 
    if ((pathlen = strlen(path)) > sizeof(saun->sun_path)) {
-      Warn2("unix socket address "F_Zu" characters long, truncating to "F_Zu"",
+      Error2("unix socket address "F_Zu" characters long, max length is "F_Zu"",
 	    pathlen, sizeof(saun->sun_path));
    }
    strncpy(saun->sun_path, path, sizeof(saun->sun_path));	/* ok */
